src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+#include "pal_types.h"
+
+#ifdef  GSSFW
```

Comment:
@vijaykota No worries :) but this still doesn't look to be fixed :(

@eerhardt There isn't right now, but by including things in headers, it makes it difficult for anything to include that header file. The convention to prevent these problems is to forward declare types in headers and only include what is strictly necessary, then in the CPP do as you wish since nobody will include those.